Formula/geos.rb

```diff
@@ -31,7 +31,8 @@ def install
     system "./configure", "--disable-dependency-tracking",
                           "--prefix=#{prefix}",
                           "--enable-python",
-                          "PYTHON=#{Formula["python@3.9"].opt_bin}/python3"
+                          "PYTHON=#{Formula["python@3.9"].opt_bin}/python3",
+                          "--disable-inline"
```

Comment:
This should only happen on ARM though, please check how other formulae use an `args` array to achieve this.

Reply:
Thanks. I fixed it